pages/wiki/documentation.md

```diff
@@ -32,6 +32,7 @@ layout: documentation
   <li><a href="{{rel 'wiki/ble-profiles'}}">Bluetooth Low Energy Profiles</a></li>
   <li><a href="{{rel 'wiki/conferences'}}">Conferences</a></li>
   <li><a href="{{rel 'wiki/porting-guide'}}">Porting Guide</a></li>
+  <li><a href="{{rel 'wiki/bitbake'}}">Working with bitbake</a></li>
```

Comment:
I haven't read the content yet because I understand you still need to address MagneFire's comments and you have more stuff you wanted to do but just a general note, this feels like this will have a lot of duplicated content with the existing OpenEmbedded page just below. Have you considered consolidating the two pages ? I feel that duplicating the information will only lead to fragmentation and a well structured document should be able to start with a high level description suiting everyone followed by a deeper dive suiting those who are interested in it deeper.

Reply:
You're right.  I've combined the pages and it's much nicer now, without all of the duplication.